x/schedule/cutoff.go

```diff
@@ -72,12 +72,17 @@ func (ct *CutoffTimes) register(tz string, timestamp string) error {
 	}
 
 	var zone string
+	var location *time.Location
 	if tz != "" {
 		zone = fmt.Sprintf("CRON_TZ=%s", tz)
+		l, _ := time.LoadLocation(tz)
+		location = l
+	} else {
+		location = time.UTC
```

Comment:
It'd be nice to have a log msg after this block to indicate which timezone we're using

Reply:
We can do that.

Reply:
https://github.com/moov-io/paygate/pull/636